Webex Connect Flows/v3.4/Template/Media Specific Workflows/Google Business Messages/README.md

```diff
@@ -0,0 +1,48 @@
+# Overview of Google Business Messages template flows
+
+- This document covers the purpose and configuration details for the below template flows for Google Business Messages channel
+  - Basic Inbound flow
+  - Inbound flow with suggestions
+- Inorder to publish the flows you will need assets configured for Google Business Messages channel you can find the documentation for asset configuration in https://help.imiconnect.io/docs/google-business-messages
+
+## Overview of Google Business Messages Inbound flow
+
+- This workflow will be triggered for every inbound customer message from the Google Business Messages Asset.
+- This flow contains the basic steps to resolve a conversation and queuing it, when a message is received on the Google Business Messages Asset 
+    ![GBMInboundFlow](../../../images/GBMInboundFlow.jpeg)
+- Google Business Messages can be used to send Text Messages, Images, Rich Cards, and Carousel Cards to your end customers.
+## Inbound flow with suggestions
+
+- This workflow will be triggered for every inbound customer message from the Google Business Messages Asset.
+- When a message is received on the Google Business Messages Asset the flow will offer a few text suggestions on the type of service offered and get the customer response before queuing the contact to live agent for further deliberation
```

Comment:
queuing the contact to live agent for further handling

Reply:
done